src/main/java/com/jfrog/ide/common/ci/BuildGeneralInfo.java

```diff
@@ -37,6 +47,18 @@ public Vcs getVcs() {
         return vcs;
     }
 
+    public BuildGeneralInfo buildName(String buildName) {
+        this.buildName = buildName;
+        updateComponentID();
```

Comment:
I think it will be more elegant to override the getComponentId method:
```java
@Override
public String getComponentId() {
    return this.buildName + ":" + this.buildNumber;
}
```